.github/workflows/test.yml

```diff
@@ -71,7 +84,9 @@ jobs:
       - run: cp src/codecov.yml .
 
       - name: Rosdep
-        run: DEBIAN_FRONTEND=noninteractive rosdep update && rosdep install --from-paths src --ignore-src --rosdistro rolling -y
+        run: |
+          . /opt/ros/$ROS_DISTRO/setup.sh
+          DEBIAN_FRONTEND=noninteractive rosdep update && rosdep install --from-paths src --ignore-src --rosdistro rolling -y
```

Comment:
the source uses the $ROS_DISTRO variable, but the install uses rolling, can we use the $ROS_DISTRO there?

Reply:
Yes good point